integration-tests/modules/__tests__/inventory/index.spec.ts

```diff
@@ -367,6 +367,115 @@ medusaIntegrationTestRunner({
             }),
           ])
         })
+
+        it("should fail to create a location level for an inventory item", async () => {
+          const [{ id: inventoryItemId }] = await service.list({})
+
+          const error = await api
+            .post(
+              `/admin/inventory-items/${inventoryItemId}/location-levels`,
+              {
+                location_id: "{location1.id}",
+                stocked_quantity: 10,
+              },
+              adminHeaders
+            )
+            .catch((error) => error)
+
+          expect(error.response.status).toEqual(404)
+          expect(error.response.data).toEqual({
+            type: "not_found",
+            message: "Stock locations with ids: {location1.id} was not found",
+          })
+        })
+      })
+
+      describe.skip("Create inventory items", () => {
+        it("should create inventory items", async () => {
+          const createResult = await api.post(
+            `/admin/products`,
+            {
+              title: "Test Product",
+              variants: [
+                {
+                  title: "Test Variant w. inventory 2",
+                  sku: "MY_SKU1",
+                  material: "material",
+                },
+              ],
+            },
+            adminHeaders
+          )
+
+          const inventoryItems = await service.list()
+
+          expect(inventoryItems).toHaveLength(0)
+
+          const response = await api.post(
+            `/admin/inventory-items`,
+            {
+              sku: "test-sku",
+              variant_id: createResult.data.product.variants[0].id,
+            },
+            adminHeaders
+          )
+
+          expect(response.status).toEqual(200)
+          expect(response.data.inventory_item).toEqual(
+            expect.objectContaining({
+              sku: "test-sku",
+            })
+          )
+        })
+
+        it("should attach inventory items on creation", async () => {
+          const createResult = await api.post(
+            `/admin/products`,
+            {
+              title: "Test Product",
+              variants: [
+                {
+                  title: "Test Variant w. inventory 2",
+                  sku: "MY_SKU1",
+                  material: "material",
+                },
+              ],
+            },
+            adminHeaders
+          )
+
+          const inventoryItems = await service.list()
```

Comment:
Let's try to use the http endpoints in the integration tests so as not to have to rely on directly accessing the module.